src/components/index.js

```diff
@@ -1 +1,3 @@
-export * from './selectable-dropdown';
+/* eslint-disable import/prefer-default-export */
+export { default as SelectableDropdown } from './SelectableDropdown';
+export { default as PostActionsBar } from '../discussions/posts/post-actions-bar/PostActionsBar';
```

Comment:
This will not work, and I don't think it's valid export syntax. You can read more here: https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#re-exporting_aggregating

Reply:
Thank you for raising this point. I will resolve all these reviews.